hbase-server/src/main/java/org/apache/hadoop/hbase/master/HbckChore.java

```diff
@@ -134,7 +135,7 @@ protected synchronized void chore() {
       loadRegionsFromInMemoryState();
       loadRegionsFromRSReport();
       try {
-        loadRegionsFromFS();
+        loadRegionsFromFS(scanForMergedParentRegions());
```

Comment:
nit: maybe call `scanForMergedParentRegions()` inside `loadRegionsFromFs`, for simplicity?

Reply:
Thanks @wchevreuil, this is good catch. I have pushed the patch as it is to master and branch-2.3, to keep it consistent, I will push as it is. Will put up an addendum if it is needed.